src/table/cells/handle-key-press.js

```diff
@@ -102,18 +102,20 @@ export const bodyHandleKeyPress = (
   evt,
   rootElement,
   cellCoord,
-  focusedCellCoord,
   selState,
   cell,
   selDispatch,
-  isAnalysisMode
+  isAnalysisMode,
+  setFocusedCellCoordsState,
+  handleSetMovingTop
 ) => {
   switch (evt.key) {
     case 'ArrowUp':
+      handleSetMovingTop(true);
     case 'ArrowDown':
     case 'ArrowRight':
     case 'ArrowLeft': {
-      !isCtrlShift(evt) && moveFocus(evt, rootElement, cellCoord, focusedCellCoord, selState);
+      !isCtrlShift(evt) && moveFocus(evt, rootElement, cellCoord, selState, setFocusedCellCoordsState);
```

Comment:
This is a bit weird, is this valid syntax? Because for arrowUp it runs both `handleSetMovingTop` and `moveFocus` right?

Reply:
Yes, it's totally fine, I want to run `handleSetMovingTop(true)` function specifically (only) for 'ArrowUp' case and not for other ones. since i don't put `break` for it will will run that function for that case and JS interpreter will move to other cases until it reaches to the break statement. 

it is the same concept of putting multiple cases together and running some part of code for all.

Reply:
cool, didn't know that. Looks like you need a comment thoough https://eslint.org/docs/rules/no-fallthrough